use_cases/gretel.json

```diff
@@ -71,6 +71,24 @@
         "bytes": 405000
       }
     },
+    {
+      "gtmId": "use-case-downstream-accuracy",
+      "title": "Synthesize data + evaluate ML performance",
```

Comment:
very small thing, can we add "[NEW]" to the card titles?

Reply:
new commit